extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -169,6 +174,21 @@ export class Placement {
     }
   }
 
+  /**
+   * Add custom amp-analytics emement
+   */
+  addCustomAmpAnalytics_() {
+    if (this.customAnalytics_) {
+      const doc = this.ampdoc.win.document;
+      const customAmpAnalytics = createElementWithAttributes(
+        doc,
+        'amp-analytics',
+        this.customAnalytics_
+      );
+      this.getAdElement().appendChild(customAmpAnalytics);
```

Comment:
why do we need to create both here and in anchor-ad-strategy?

Reply:
@calebcordry When I implement the feature I've seen different flows, there is no mention of 'placement' inside 'ad-anchor-strategy', so I see the needed of add the amp-analytics in both, to the amp-analytics be added inside the 'amp-ad' properly